internal/commands/app/create_inputs.go

```diff
@@ -98,32 +111,71 @@ func (i *createInputs) resolveDirectory(wd string) (string, error) {
 	if appOK {
 		return "", errProjectExists{fullPath}
 	}
+	ui.Print(terminal.NewWarningLog("Directory './%s' already exists, writing app contents to that destination may result in file conflicts.", i.Directory))
+	proceed, err := ui.Confirm("Would you still like to write app contents to './%s'? ('No' will prompt you to provide another destination)", i.Directory)
+	if err != nil {
+		return "", err
+	}
+	if !proceed {
+		var newAppName string
+		if err := ui.AskOne(&newAppName, &survey.Input{Message: "App Name"}); err != nil {
```

Comment:
whats the reasoning behind updating `Name` here too?  I'm not fully thinking this one through yet, but can we allow the user to create an app with one name but in a directory named something else?  It _feels_ like that should be ok, but we're firmly in the land of nuances and caveats here...

Reply:
Yeah I agree, was just trying to remove confusion, since a majority of the time directory == appName. If you think we should leave it alone, I'm good with that.

Reply:
Going with only updating directory